vite.config.mjs

```diff
@@ -142,72 +143,70 @@ function copyToPkgDirPlugin() {
     name: 'copy-patterns-plugin',
     apply: 'build',
     async buildStart(opts) {
-      if (!fs.existsSync(pkg_dir)) {
-        serverPackageFolders.forEach((serverPackageFolder) => {
-          fs.mkdirSync(serverPackageFolder, { recursive: true })
-        })
-        fs.mkdirSync(pkg_dir + '/dist')
-        fs.mkdirSync(pkg_dir + '/src/language', {
-          recursive: true,
-        })
-        fs.mkdirSync(pkg_dir + '/src/language/providers', {
-          recursive: true,
-        })
-        fs.mkdirSync(pkg_dir + '/src/language/providers/intellisense/', {
-          recursive: true,
-        })
-        fs.mkdirSync(pkg_dir + '/src/launchWizard', {
-          recursive: true,
-        })
-        fs.mkdirSync(pkg_dir + '/src/styles', {
-          recursive: true,
-        })
-        fs.mkdirSync(pkg_dir + '/src/tdmlEditor', {
-          recursive: true,
-        })
-      }
+      ;[
+        pkg_dir + '/dist',
+        pkg_dir + '/src/language',
+        pkg_dir + '/src/language/providers',
+        pkg_dir + '/src/language/providers/intellisense/',
+        pkg_dir + '/src/launchWizard',
+        pkg_dir + '/src/styles',
+        pkg_dir + '/src/tdmlEditor',
+        ...serverPackageFolders,
+      ].forEach((folder) => {
+        if (!fs.existsSync(folder)) {
+          fs.mkdirSync(folder, { recursive: true })
+        }
+      })
 
       for (const { from, to } of patterns) {
-        from.includes('.')
+        fs.statSync(from).isFile()
           ? fs.copyFileSync(from, to)
           : fs.cpSync(from, to, { recursive: true })
       }
     },
   }
 }
-const shouldMinify = process.env.MINIFY === '1'
-export default defineConfig({
-  resolve: {
-    alias: {
-      ...localModuleAliases,
-    },
-    extensions: ['.ts', '.js'],
-  },
 
-  build: {
-    sourcemap: true,
-
-    minify: shouldMinify ? 'esbuild' : false,
-
-    outDir: path.resolve(__dirname, '../dist/package/dist/ext'),
-    emptyOutDir: true,
+const shouldMinify = process.env.MINIFY === '1'
```

Comment:
Does this want to be controlled via production vs development mode rather than an env variable? Or is the MINIFY env variable the standard way to do this? Should we be enabling MINIFY as part of the release-candidate script?